packages/legacy/core/App/localization/en/index.ts

```diff
@@ -248,7 +248,7 @@ const translation = {
     "BiometricsNotProvided": "Biometrics not provided, you may use PIN to load the wallet.",
     "BiometricsError": "Biometrics were not successful.",
     "BiometricsErrorEnterPIN": "Please enter your wallet PIN.",
-    "LockedOut": "You locked out after 5 minutes of inactivity.",
+    "LockedOut": "You were locked out after ## minute(s) of inactivity.",
```

Comment:
There's an example of how to do this without using `String.prototype.replace` if you check the "MinutesAgo" entry.

Reply:
swapped out with established pattern